Javascript/q44.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q44</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    th {
+      text-align: left;
+    }
+  </style>
+
+  <script>
+    function myFunction() {
+      const length = document.querySelector("#demo0").value;
+      const count = document.querySelector("#demo1").value;
+      const charList = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@#$%&*-_';
```

Comment:
You should put this const outside the function definition. Having this string here is inefficient for the computer as it has to recreate and allocate memory for this function each time the function runs. That takes time. It might just be nanoseconds in this case but inefficiencies like this add up in big programs. By keeping it outside the function, the variable will only be created and allocated into memory once when the page starts

Reply:
ok